README.md

```diff
@@ -20,19 +20,69 @@ for how to create a new repository from this template on GitHub.
 
 ## Running Acceptance Tests
 
-Make sure to install the plugin with `go build .` and to have Packer installed locally.  
-Then source the built binary to the plugin path with `cp packer-plugin-scaffolding ~/.packer.d/plugins/packer-plugin-scaffolding`  
-Once everything needed is set up, run:  
+Make sure to install the plugin with `go build .` and to have Packer installed locally.
+Then source the built binary to the plugin path with `cp packer-plugin-scaffolding ~/.packer.d/plugins/packer-plugin-scaffolding`
+Once everything needed is set up, run:
 ```
 PACKER_ACC=1 go test -count 1 -v ./... -timeout=120m
 ```
 
 This will run the acceptance tests for all plugins in this set.
 
-## Requirements
+## Registering Documentation on Packer.io
+
+Documentation for a plugin is maintained within the `docs` directory and served on GitHub.
+To include plugin docs on Packer.io a GitHub workflow has been added to this template repository that
+can be used generate a documentation structure that can be consumed remote by Packer.io.
+See the workflow [generated-docs-artifacts.yml](.github/workflows/generate-dos-artifacts.yml)
+
+After merging the generated files to the default branch for the plugin repository: main or master
+
+A one time pull-request will needs to be opened against [hashicorp/packer](https://github.com/hashicorp/packer) to register the plugin docs.
+This is done by adding the block below for the respective plugin to the file `website/data/docs-remote-navigation.js`.
```

Comment:
we should come back and make `website/data/docs-remote-navigation.js` a real link once it's merged with master on Packer